.yamato/publish.yml

```diff
@@ -0,0 +1,29 @@
+{% metadata_file .yamato/environments.yml %}
+
+---
+{% for variant in package_variants %}
```

Comment:
We can do away with the variants now - I can't think of any use for variants in the Perception package.